django-backend/fecfiler/transactions/views.py

```diff
@@ -234,16 +235,26 @@ def get_previous(
                 Q(contact_2__candidate_state=state),
                 Q(contact_2__candidate_district=district),
             )
+
+        original_transaction = None
         if transaction_id:
-            transaction = self.get_queryset().get(id=transaction_id)
+            original_transaction = self.get_queryset().get(id=transaction_id)
             query = query.filter(
-                Q(created__lt=transaction.created) | ~Q(date=date)
+                Q(created__lt=original_transaction.created) | ~Q(date=date)
             )
 
         query = query.order_by("-date", "-created")
         previous_transaction = query.first()
 
         if previous_transaction:
+            if original_transaction and (
```

Comment:
I don't think we need to be changing anything for this to work.  The only data that matters from the existing transaction is its created date.  The rest is entirely derived from the query params